rhizome/host/bin/setup-node-exporter.rb

```diff
@@ -0,0 +1,45 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../../common/lib/util"
+require "fileutils"
+
+unless (version = ARGV.shift)
+  fail "No version provided"
+end
+
+file_name = case r("common/bin/arch").strip
+when "arm64"
+  "node_exporter-#{version}.linux-arm64"
+when "x64"
+  "node_exporter-#{version}.linux-amd64"
+else
+  raise "Unsupported architecture: #{arch}"
+end
```

Comment:
We have a helper for it, the one @fdr loves the most.


```suggestion
file_name = "node_exporter-#{version}.linux-#{Arch.render(x64: "amd64", arm64: "arm64")}"
```

Reply:
that's way cleaner thanks!